helpdesk_mgmt/migrations/14.0.1.5.1/post-migration.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2022 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(
+        env.cr, "helpdesk_mgmt", "migrations/14.0.1.5.1/noupdate_changes.xml"
```

Comment:
In this case, I think it's better to do a replace in the string instead of losing whole template for not resetting possible customizations.

Reply:
Done!